source/appModules/notepad++.py

```diff
@@ -0,0 +1,27 @@
+#appModules/notepad++.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2017 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+"""App module for Notepad++
+"""
+
+import appModuleHandler
+from NVDAObjects import NVDAObject
+
+class NotepadPlusPlusScintillaEdit(NVDAObject):
+
+	def initOverlayClass(self):
+		pass
+
+	def shouldAcceptShowHideCaretEvent(self):
+		return False

```

Comment:
If this were an attribute as above, this could just be: `shouldAcceptShowHideCaretEvent = False`